modules/oe_theme_content_project/src/Plugin/ExtraField/Display/PercentageExtraField.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_content_project\Plugin\ExtraField\Display;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Cache\CacheableMetadata;
+
+/**
+ * Extra field displaying eu budget percentage on projects.
+ *
+ * @ExtraFieldDisplay(
+ *   id = "oe_theme_content_project_percentage",
+ *   label = @Translation("Percentage"),
+ *   bundles = {
+ *     "node.oe_project",
+ *   },
+ *   visible = true
+ * )
+ */
+class PercentageExtraField extends ProjectExtraFieldBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getLabel() {
+    // Add space as label so the value aligns with the other fields.
+    return ' ';
```

Comment:
Mh... don't we have any other way of aligning it with other fields? I'd rather not use empty labels for this.

Reply:
Fixed using CSS